Build/17.0/packages.config

```diff
@@ -85,4 +85,5 @@
   <package id="Microsoft.VisualStudio.RpcContracts" version="17.0.50-preview-0002-0005" />
   <package id="Microsoft.DiaSymReader.Pdb2Pdb" version="1.1.0-beta2-21181-01" />
   <package id="Microsoft.TestPlatform.ObjectModel" version="17.4.0-preview-20221003-03" />
+  <package id="Microsoft.DiagnosticsHub.SDK" version="17.13.35618.1" />
```

Comment:
Is this unrelated?

Reply:
Shoot yes, this made it up by mistake, alone with the nuget.config. I'll take them out.